src/styles/index.scss

```diff
@@ -2,2 +1,69 @@
+@import 'variables';
+
+html {
+  font-family: Arial, sans-serif;
+}
+
+* {
   margin: 0;
+  padding: 0;
+}
+
+body {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+}
+
+@each $day, $index in $days {
+  $index: $index + 6;
```

Comment:
The `$index` variable is being reassigned on each iteration which is not a good practice in SCSS. It's better to define a new variable if you need to adjust the value of `$index` for the calculation.